web/pages/api/v0/market/[id].ts

```diff
@@ -3,12 +3,17 @@ import { Bet, listAllBets } from 'web/lib/firebase/bets'
 import { listAllComments } from 'web/lib/firebase/comments'
 import { getContractFromId } from 'web/lib/firebase/contracts'
 import { applyCorsHeaders, CORS_UNRESTRICTED } from 'web/lib/api/cors'
+import { assertHTTPMethod } from 'web/lib/api/validation'
 import { FullMarket, ApiError, toLiteMarket } from '../_types'
 
 export default async function handler(
   req: NextApiRequest,
   res: NextApiResponse<FullMarket | ApiError>
 ) {
+  if (!assertHTTPMethod(req, res, 'GET')) {
+    return
+  }
```

Comment:
I'm concerned about these `GET` methods that aren't proxying to Firebase because they should also support `HEAD` and `OPTIONS` requests.

Reply:
I definitely care about correctly interpreting the method and producing appropriate errors, I just don't want to not support functionality we want.